source/tests/system/nirfmxlte_driver_api_tests.cpp

```diff
@@ -54,12 +55,30 @@ InitializeResponse init(const client::StubPtr& stub, const std::string& model)
   return client::initialize(stub, "FakeDevice", options);
 }
 
+nirfsa_grpc::InitWithOptionsResponse init_rfsa(const nirfsa_client::StubPtr& stub, const std::string& resource_name)
+{
+  return nirfsa_client::init_with_options(stub, resource_name, false, false, "Simulate=1, DriverSetup=Model:5663E");
+}
+
 TEST_F(NiRFmxLTEDriverApiTests, Init_Close_Succeeds)
 {
   auto init_response = init(stub(), PXI_5663E);
   auto session = init_response.instrument();
-  // TODO: use EXPECT_SUCCESS here when LTE has get_error and get_error_string
-  EXPECT_RESPONSE_SUCCESS(init_response);
+  EXPECT_SUCCESS(session, init_response);
+
+  auto close_response = client::close(stub(), session, 0);
+
+  EXPECT_RESPONSE_SUCCESS(close_response);
```

Comment:
Note: In both tests, I'm using EXPECT_RESPONSE_SUCCESS for the `close_response` to match what's done for the similar tests in other personalities. 

The reasoning to use this over EXPECT_SUCCESS is captured in an [Instr driver test](https://github.com/ni/grpc-device/blob/85c6286076e8c42aa411eb40c67ee96710b4c053/source/tests/system/nirfmxinstr_driver_api_tests.cpp#L131): ""Don't use EXPECT_STATUS because that checks the error text, which can report stale errors from previous tests".